tests/ops/test_channel_ops.py

```diff
@@ -433,6 +433,7 @@ def test_p_arbitrary(self, p, tol):
         expected_K1 = np.sqrt(p) * Z
         assert np.allclose(op(p, wires=0).kraus_matrices()[1], expected_K1, atol=tol, rtol=0)
 
+    # !TODO: figure out why specifically PhaseFlip(0) not working
```

Comment:
This seems something that should be addressed

Reply:
This is already addressed by the fix to `probs.py`; but according to @lillian542  we will have to delay the fix for future discussion after new year

Reply:
This test point at `agnle=0` will be omitted for now